src/Symfony/Component/Validator/Constraints/PasswordStrengthValidator.php

```diff
@@ -36,7 +36,7 @@ public function validate(#[\SensitiveParameter] mixed $value, Constraint $constr
             return;
         }
 
-        if (!\is_string($value)) {
+        if (!\is_string($value) && !$value instanceof \Stringable) {
             throw new UnexpectedValueException($value, 'string');
         }
         $passwordStrengthEstimator = $this->passwordStrengthEstimator ?? self::estimateStrength(...);
```

Comment:
The value may be casted to `(string)` when calling `$passwordStrengthEstimator($value)`. We don't know how this is handled by the `$passwordStrengthEstimator` callable.

Reply:
indeed, done